samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/BookingDialog.java

```diff
@@ -36,22 +42,22 @@ public BookingDialog() {
             this::confirmStep,
             this::finalStep
         };
-        addDialog(new WaterfallDialog(WaterfallDialog.class.getName(), Arrays.asList(waterfallSteps)));
+        addDialog(new WaterfallDialog("WaterfallDialog", Arrays.asList(waterfallSteps)));
```

Comment:
We had this in a constant before, following [TS](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/typescript_nodejs/13.core-bot/src/dialogs/bookingDialog.ts#L20). Not super mandatory though

Reply:
You're right, but it supposed to be the class name, so i think it's OK the way we have it

Reply:
Agree

Reply:
We can leave this as is right now